files/default/agent.py

```diff
@@ -236,7 +224,7 @@ def send(self):
                 now = long(time.mktime(datetime.now().timetuple()))
                 headers = {'content-type': 'application/json'}
                 payload = {}
-                payload["num-gpus"] = count_num_gpus()
+                payload["num-gpus"] = devices.get_num_gpus()
```

Comment:
It is important that 'get_num_gpus' does not raise any exception that could kill the heartbeat. Otherwise I'm OK with PR

Reply:
This should be reliable enough: https://github.com/logicalclocks/hops-util-py/blob/bbb4232036ee8057d57b4d472a49b715cc22771c/hops/devices.py#L115